imports/plugins/core/orders/server/no-meteor/mutations/placeOrder.js

```diff
@@ -1,4 +1,5 @@
 import SimpleSchema from "simpl-schema";
+import R from "ramda";
```

Comment:
Single character variables are not allowed by Reaction style rules. I'd recommend just using the standard array `.find` in this case, though.